src/lib.rs

```diff
@@ -74,19 +74,23 @@
 //!
 extern crate bevy_proto_derive;
 
-pub mod components;
+mod components;
+pub use components::ProtoComponent;
```

Comment:
This might be confusing for a user who is trying to use it as a the derive macro (in which case, they need to write `bevy_proto::prelude::ProtoComponent`).

I think we'd either need to export the derive here too or keep this in the prelude (I'm leaning more towards the former).